resources/execute_testing.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install edX app on emulator 
+APK=$APK_PATH/"*.apk"
```

Comment:
I think you will need to set APK_PATH here. I am not sure this environment variable will be present from the the previous script

Reply:
This environment variable is available from the the previous jenkinsfile

Reply:
Ok, great. I forgot it was set in the Jenkinsfile